### PR TITLE
Use `vmware-vmx` builder to build OpenBSD box for arm64

### PR DESCRIPTION
**Pull request by dentarg**
Uses the `vmware-vmx` builder as there is no ISO files built for arm64 releases currently.

* https://developer.hashicorp.com/packer/integrations/hashicorp/vmware/latest/components/builder/vmx
* https://github.com/hashicorp/packer-plugin-vmware

Closes https://github.com/twingly/boxes/issues/39 as it changes the default mirror value.

Closes https://github.com/twingly/boxes/issues/38 too.

### CODE REVIEW

**Comment by dentarg** [comment]
> no ISO files built for arm64 releases currently

There probably will be for OpenBSD 7.5: https://marc.info/?l=openbsd-misc&m=169808954409925&w=2

There is ISO for snapshots: https://ftp.lysator.liu.se/pub/OpenBSD/snapshots/arm64/

**Comment by dentarg** [comment]
I've uploaded the box to https://app.vagrantup.com/starkast/boxes/openbsd-7.1-arm64

**Comment by dentarg** [comment]
Built OpenBSD 7.2 arm64 for VMware Fusion too: https://app.vagrantup.com/starkast/boxes/openbsd-7.2-arm64/versions/1.0.0

File: openbsd/README.md [hunk 1]
```diff
@@ -2,9 +2,23 @@
...
 1. Verify the checksum of the new ISO [with signify]
 1. Update the template file
     1. Update the version
     1. Update the checksum
 
 [with signify]: https://www.openbsd.org/faq/faq4.html#Download
+
+# VMware Fusion
+
+Used to build `amd64` boxes.
```
Comment (vikiv480) [inline openbsd/README.md:18]:
```suggestion
Used to build `arm64` boxes.
```

**Review by vikiv480 — APPROVED** [review]
I managed to build an OpenBSD 7.3 box. Tested an internal Ansible playbook and I got it to work with some minor changes to the network setup. 

Thank you, LGTM! 🎉

**Comment by dentarg** [comment]
What network setup changes did you have to do? Where?

**Comment by vikiv480** [comment]
>What network setup changes did you have to do? Where?

In our Ansible setup the internal IP-address was hard-coded to the [default value for Virtualbox](https://www.virtualbox.org/manual/ch06.html#:~:text=the%20first%20card%20is%20connected%20to%20the%20private%20network%2010.0.2.0), I had a different default IP-address defined in `/Library/Preferences/VMware\ Fusion/vmnet8/dhcpd.conf`. I temporarily hard-coded the new IP-address in Ansible. 

I also encountered [this issue](https://gist.github.com/muhammadghazali/6c2b8c80d5528e3118613746e0041263) during the setup of Nginx where I had to increase the `server_names_hash_bucket_size` from 32 to 64.

I didn't make any changes to `twingly/boxes` when building the box (other than changing the version to 7.3), I just had to configure our Ansible setup 🙂

**Pull request merged**